app/templates/index.html

```diff
@@ -0,0 +1,39 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+  <div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+
+  <div class="input">
+    <form action="/" method="POST">
+      <input type="text" name="item" id="item" required>
+      <button type="submit" id="submit">Add Item</button>
+    </form>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Things To Do!</h2>
+      <ul class="list-group-item">
+        <!-- List the todo items here -->
+          {% for item in view_model.items %}
+            <li>{{ item.title }} - {{item.status}} 
+              <table>
+                {% if item.status == 'To Do' %}
+                  <tr>
+                    <form class="completed" action="/update/{{item.id}}" method="POST"><input type="submit" value="Complete" id="{{item.title}}_complete"></form>
```

Comment:
You should use item.id rather than item.title 
- ids should be unique but you do not enforce that for `item.title`
- Technically, HTML ids should not contain spaces